samples/ai-agent-with-mcp/teardown.sh

```
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# teardown.sh — tear down all containers started by setup.sh
+
+set -uo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+cd "$SCRIPT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)